test_files/clang-c/Index.h

```diff
@@ -5206,7 +5206,7 @@ CINDEX_LINKAGE CXResult clang_findIncludesInFile(CXTranslationUnit TU,
                                               CXCursorAndRangeVisitor visitor);
 
 #ifdef __has_feature
-#  if __has_feature(blocks)
+#  if 0 && __has_feature(blocks)
```

Comment:
I figured out a way to call Clang blocks without language support using a library solution [1], if we would like to translate Clang blocks.

[1] https://github.com/jacob-carlborg/druntime/blob/550edd0a64f0eb2c4f35d3ec3d88e26b40ac779e/src/core/stdc/clang_block.d